QUARKUS-744.md

```diff
@@ -0,0 +1,31 @@
+# QUARKUS-744 - Non application endpoints redirections
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-744
+
+The goal is to verify non-application redirections, in order to don't break users common use-cases.
+
+## Scope of the testing
+
+The following non-application endpoint will be covered:
+
+```
+- /metrics
+     - /metrics/vendor
+     - /metrics/application
+    - /metrics/base
+- /health
+   - /health/live
+   - /health/ready
+  - /health/well
+  - /health/group
+- /openapi
+- /swagger-ui
+```
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works on Openshift (JVM and NATIVE mode)
+
```

Comment:
Add section which describes which "clients" will interact with the application.

"clients" can be java based like apache httpclient, vertx webclient, mp rest-client, etc.
But also health checks in OpenShift, Kubernetes, google cloud, amazon aws, custom health check code.

I have seen that access log got polluted with redirect entries for `/health` and `/metrics`, that needs to checked too.